src/segment-loader.js

```diff
@@ -859,7 +885,18 @@ export default class SegmentLoader extends videojs.EventTarget {
   loadSegment_(segmentInfo) {
     this.state = 'WAITING';
     this.pendingSegment_ = segmentInfo;
-    this.trimBackBuffer_(segmentInfo);
+
+    // Chrome has a hard limit of 150MB of
```

Comment:
Not too big of a deal, but I think this comment still makes sense to be in the `trimBackBuffer` function, since that is where the logic of how much to clear happens. `loadSegment` just needs to know to call `trimBackBuffer`